src/Gui/PythonWrapper.cpp

```diff
@@ -563,6 +563,32 @@ QGraphicsItem* PythonWrapper::toQGraphicsItem(PyObject* pyPtr)
     return nullptr;
 }
 
+QGraphicsItem* PythonWrapper::toQGraphicsItem(const Py::Object& pyobject)
+{

```

Comment:
I suggest to avoid the code-duplication and instead write: `return toQGraphicsItem(pyobject.ptr());`

Reply:
Thanks. Done.